WORKSPACE

```diff
@@ -1,45 +1,2 @@
-maven_jar(
-    name = "jsr305",
-    artifact = "com.google.code.findbugs:jsr305:3.0.1",
-)
-
-maven_jar(
-    name = "plexus_utils",
-    artifact = "org.codehaus.plexus:plexus-utils:jar:3.0.24",
-)
-
-maven_jar(
-    name = "plexus_component_annotations",
-    artifact = "org.codehaus.plexus:plexus-component-annotations:1.7.1",
-)
-
-maven_jar(
-    name = "plexus_interpolation",
-    artifact = "org.codehaus.plexus:plexus-interpolation:1.24",
-)
-
-maven_jar(
-    name = "guava",
-    artifact = "com.google.guava:guava:20.0",
-)
-
-maven_jar(
-    name = "maven_artifact",
-    artifact = "org.apache.maven:maven-artifact:3.5.0",
-)
-
-# Testing
-maven_jar(
-    name = "org_mockito",
-    artifact = "org.mockito:mockito-all:1.9.5",
-)
-
-maven_jar(
-    name = "org_junit",
-    artifact = "junit:junit:4.4",
-)
-
-maven_jar(
-    name = "com_google_truth",
-    artifact = "com.google.truth:truth:0.30",
-)
+load('//:generate_workspace.bzl', 'generated_maven_jars')
+generated_maven_jars()
```

Comment:
Thanks. Are you using google's linter before you push code onto github? Any OSS alternatives? I really relied on it to avoid nits.